### DISCUSSION TITLE
Gantt chart & issue start & end dates

### DISCUSSION THREAD

**Discussion by smajkows** [community/community#10704] [Projects and Issues]
Hi this project functionality seems great, the last piece we would really need to move our full planning and workload for scheduling tasks and managing development pipeline would be if we could set start&end dates on issues and have a Gantt chart style view to see how tasks are progressing in a visual timeline.

Thank you for your consideration!

**Comment by JohnnyKB** [comment]
This would help us use Github Issues as a unique tool for project management.

**Comment by erikr** [comment]
Agree. Duplicate of https://github.com/github/feedback/discussions/6553.
